distribution/docs/src/main/resources/content/_data/federation.adoc

```diff
@@ -1,76 +0,0 @@
-:title: Federation
-:type: federationIntro
-:status: published
-:summary: Federation for Integrators
-:order: 00
-
-.Federation
-[ditaa, catalog_architecture_federation, png, ${image-width}]
```

Comment:
❓ did we completely replace or remove these diagrams?

Reply:
I removed them. but I'm open to arguments for putting them back in. I think there might be a case for the second one, if we want to explain it better as it doesn't seem to add much understanding here and I don't know if there's a better place for it elsewhere

Reply:
I'm good for now, since we haven't gotten a lot of feedback on the diagrams